### PR TITLE
Drop ice35

### PR DESCRIPTION
**Pull request by jburel**
# What this PR does

Remove support for Ice 3.5.

# Testing this PR

Check that Travis is green.

# Related reading

https://trello.com/c/aCLtBiHl/17-ice-35-omero

### CODE REVIEW

**Comment by mtbc** [comment]
Maybe can delete `components/tools/OmeroWeb/requirements-py27-*ice35.txt`.

**Comment by jburel** [comment]
Thanks I thought I got rid of it :-)

**Comment by mtbc** [comment]
Probably outwith this PR but I wonder if we still need to set Ice.Default.EncodingVersion to 1.0.

**Comment by joshmoore** [comment]
There should be a note somewhere about when we can drop that; could well be it's now.

**Comment by jburel** [comment]
https://doc.zeroc.com/ice/3.7/the-ice-protocol/protocol-and-encoding-versions

File: components/tools/OmeroPy/src/omero/clients.py [hunk 1]
```diff
@@ -187,39 +187,30 @@ def _initData(self, id):
...
-                self._optSetProp(id, "IceSSL.Ciphers", "NONE (DH_anon.*AES)")
-            else:
-                self._optSetProp(id, "IceSSL.Ciphers", "ADH")
+        if sys.platform == "darwin":
+            self._optSetProp(id, "IceSSL.Ciphers", "NONE (DH_anon.*AES)")
         else:
             self._optSetProp(id, "IceSSL.Ciphers", "ADH")
+
         self._optSetProp(id, "IceSSL.VerifyPeer", "0")
         self._optSetProp(id, "IceSSL.Protocols", "tls1")
```
Comment (mtbc) [inline components/tools/OmeroPy/src/omero/clients.py:208]: Wondering if this should be `TLS1_2`!

**Comment by mtbc** [comment]
Thank you, looks to me to be good to merge.

File: components/tools/OmeroPy/src/omero/clients.py (hunk 1 above)
Comment (jburel) [reply components/tools/OmeroPy/src/omero/clients.py:208]: This will require testing

**Pull request merged**